api/src/main/java/com/basistech/rosette/api/RosetteAPI.java

```diff
@@ -1790,6 +1758,16 @@ private String headerValueOrNull(Header header) {
                     if (ecHeader != null) {
                         errorResponse.setCode(ecHeader);
                     }
+                    if (429 == status) {
+                        String concurrencyMessage = "Concurrency limit exceeded. "
```

Comment:
Please modify:

You have exceeded your plan's limit on concurrent calls.

instead of 'Concurrency limit exceeded.'